### PR TITLE
Home work 05

### PR DESCRIPTION
**Pull request by Bilyk-VM**

### CODE REVIEW

File: 05/homework_05.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+# Перше завдання
+while True:
+    user_data = input("Введіть, будь ласка данні: ")
+    if user_data == 'exit':                                 # створимо можливість вийти з програми
+        print('Вихід з программи')
+        break
+    for char in user_data:                                  # перевіряємо кожен символ в послідовності данних
+        if char.isdigit():                                  # перевірка на введення цифри
+            char = int(char)                                # якщо цифра - переводим данні в int
+            if char == 0:                                   # перевірим чи дорівнює число нулю
```
Comment (klymenok) [inline 05/homework_05.py:11]: As far as I know `0` is an even number

File: 05/homework_05.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+                print(f'Число {char}')
+            elif char % 2 == 0:                             # перевірка на парність за допомогою ділення по модулю
+                print(f"Число {char} - парне")
+            else:
+                print(f"Число {char} - не парне")
+        elif char == ' ':                                   # якщо символ це пробіл - пропустимо ітерацію
```
Comment (klymenok) [inline 05/homework_05.py:17]: A ` ` symbol (space) could also be recognized as a symbol

File: 05/homework_05.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+            continue
+        elif not char.isalnum():                            # перевіряємо чи введений символ взагалі буква чи цифра
+            print(f"Введено символ - '{char}'")
+        else:
+            if char.istitle():                              # перевіряємо регістр літери за допомогою функції .istitle
```
Comment (klymenok) [inline 05/homework_05.py:22]: There is another function that can check if a character or a sting is uppercase: `isupper()`. It is not a mistake, but you should understand the difference: `istitle()` checks if only first letter is uppercase while other characters are lowercase (if len of string is greater than 1), when `isupper()` checks the whole string.

**Review by klymenok — APPROVED** [review]
Well done with loops and conditions. Some improvements can be made according to my comments, but it is up to you since knowledge is more important here than accuracy according to the task.

File: 05/homework_05.py (hunk 1 above)
Comment (Bilyk-VM) [reply 05/homework_05.py:11]: Sorry, I didn't know that )

File: 05/homework_05.py (hunk 2 above)
Comment (Bilyk-VM) [reply 05/homework_05.py:17]: Я хотів десь в циклі примінити команду "continue", в цьому випадку мені здалось найдоречніше це зробити.

File: 05/homework_05.py (hunk 3 above)
Comment (Bilyk-VM) [reply 05/homework_05.py:22]: Thanks, I will remember.

**Pull request merged**